service/src/test/java/bio/terra/cbas/pact/VerifyPactsAllControllers.java

```diff
@@ -204,7 +206,7 @@ public void initializeDAO() throws Exception {
     when(runSetDao.createRunSet(any())).thenReturn(1);
     when(methodDao.updateLastRunWithRunSet(any())).thenReturn(1);
     when(methodVersionDao.updateLastRunWithRunSet(any())).thenReturn(1);
-    when(runSetDao.updateStateAndRunDetails(any(), any(), any(), any(), any())).thenReturn(1);
+    when(runSetDao.updateStateAndRunSetDetails(any(), any(), any(), any(), any())).thenReturn(1);
```

Comment:
TOL: I wonder whether we should move this into the `RunSetsHelper`/`RunSetsService` layer so that the controller doesn't have to interact with the DAO directly 🤔.

Reply:
This should no longer be needed since it is now moved inside `RunSetsService`.